utils/zerovec/benches/zerovec_serde.rs

```diff
@@ -43,10 +43,16 @@ fn overview_bench(c: &mut Criterion) {
         });
     });
 
+    const ORIGINAL_CHARS: &[char] = &[
+        'ⶢ', '⺇', 'Ⱜ', '◁', '◩', '⌂', '⼅', '⏻', '⢜', '◊', 'ⲫ', '⏷', '◢', '⟉', '℞',
+    ];
+
+    let _char_zero_vec = &ZeroVec::alloc_from_slice(ORIGINAL_CHARS);
+
```

Comment:
I think you can keep this code in char_benches, it just has to be outside the bench_function calls.